workshops/lib/twilio-basic/signup.md

```diff
@@ -4,185 +4,82 @@ By the end of this tutorial, you will have a Twilio account SID and
-  - `https://dweet.io/dweet/for/4846812377`
-
-For the phone number `+1-415-238-0812` the URL would be
-`https://dweet.io/dweet/for/4152380812`
-
-To set this as your callback URL, go to the Twilio [incoming phone numbers page]
-(https://www.twilio.com/user/account/phone-numbers/incoming).
-
-> ![](img/incoming_phone_numbers_page.png)
-
-Click on your phone number
-
-> ![](img/click_on_your_phone_number.png)
-
-Scroll all the way down until you see "SMS & MMS", then in the "Request URL"
-paste in your callback URL.
-
-> ![](img/update_request_url.gif)
-
-Make sure that the right side is HTTP POST (it should be the default).
-
-> ![](img/http_post.png)
-
-Then click the "Save" button at the bottom.
-
-## Upgrading Your Account
-
-_Do not click "Upgrade Account" until the directions tell you to._
-
-> ![](img/type_twilio_texted_code.gif)
-
-Scroll all the way down:
-
-> ![](img/scroll_down.gif)
-
-Click the "Upgrade your account" button
-
-> ![](img/upgrade_account.png)
-
-You will see this screen:
-
-> ![](img/upgrade_your_account.png)
-
-### Credit Card Info
-
-Although Twilio is giving each member $100 worth of credit, you still
-need a credit card to signup for Twilio.
-
-For your convenience, we have provided a credit card to signup with below. You
-may use your own card as well (your card will not be charged if you turn off
-automatic recharging).
-
-| Field              | Value                               |
-|--------------------|-------------------------------------|
-| Credit Card Number | `4847356897256173`                  |
-| Expires            | `09/2021`                           |
-| CVV                | `917`                               |
-| Cardholder Name    | `Jonathan Leung`                    |
-| Street Address     | `200 Haverford Road, Wynnewood, PA` |
-| Zip Code           | `19096`                             |
-
-We enter this info below:
-
-> ![](img/enter_credit_card_info.gif)
-
-Leave the "Company/Project URL" and "Phone Number" fields blank
-
-> ![](img/leave_blank.png)
-
-### Entering The Promo Code
-
-Click "I have a promo code."
-
-> ![](img/i_have_a_promo_code.png)
-
-To get the promo code:
-
-  - Club members: Ask your club leader for the Twilio promo code.
-
-  - Club leaders: send an email to the email address
-    `jonathanleung1337@gmail.com` with your full name and school and ask for the
-    Twilio promo code. They will respond to you quickly.
+1. Open [https://www.twilio.com/signup](https://www.twilio.com/)
+2. Fill in your information
+  - Note that there are 3 questions like "What are you building?". Don't bother
+    answering them correctly. Just choose the first answer you see and move on.
+3. Click "Get Started" button to move to the next step
+4. Twilio then asks you to enter a phone number. Do that.
+5. Click "Text Me" and continue following remaining directions until you see
+   this screen:
+   > ![](img/finish_verification_screen.png)
```

Comment:
This should have a newline before it to separate it from the above bullet point.